src/doc/gui_manual/Operators/OperatorTypes/PICS/index.rst

```diff
@@ -0,0 +1,268 @@
+      field.
+   
+M3D-C1 3D
+    Evaluates the 3D magnetic field via a 2D poloidal 6th order polynomial and
+    1D toroidal 4th order Bezier spline.
+    
+Nek5000
+    Evaluates the 3D vector field using `Nek5000 <https://nek5000.mcs.anl.gov/>`_
+    spectral elements.
+    
+Nektar++
+    Evaluates the 3D vector field using `Nektar++ <https://www.nektar.info/>`_
+    spectral elements.
+        
+NIMROD
+    Evaluates the 3D magnetic field via a 2D poloidal 3rd order Lagrange-type
+    element and 1D toroidal finite Fourier series element (future
+    implementation).
+
+
+Integration
+^^^^^^^^^^^
+
+Specify settings for the numerical integrator. See each operator for varied
+settings.
+
+
+Integrator
+""""""""""
+
+Sets the integration scheme. Options are:
+
+Forward Euler
+    The basic Euler integration with a fixed step length. This integrator is
+    not recommended.
+
+Leap Frog
+    Simple method for numerical integration where both the position and
+    velocity are updated using a fixed step length.
+
+Runge-Kutta-Dormand-Prince (RKDP)
+    A Runge-Kutta based scheme with adaptive step size control.
+    
+Adams-Bashforth (Multi-step)
+    A linear multistep integration using a fixed step length.
+    
+Runge-Kutta
+    Traditional 4th order Runge-Kutta with a fixed step length.
+    
+M3D-C1 2D Integrator (M3D-C1 2D fields only)
+    Uses Newton's method under the hood.
+    
+         
+Step Length
+"""""""""""
+
+The length of the integration step. Most integrators use a fixed step length.
+Runge-Kutta-Dormand-Prince uses adaptive step size, which can be clipped by
+this value.
+
+
+Tolerances
+""""""""""
+
+RKDP, Adams-Bashforth, and MD3-C1 make use of the tolerance options.
+    
+RKDP
+    Two tolerances are available - The adaptive step is set such that the
+    maximum error at each step is less than the maximum between the absolute
+    tolerance and the relative tolerance times the value of the vector field at
+    the current point. The absolute tolerance can be truly absolute or relative
+    to the bounding box.
+
+Adams-Bashforth
+    NOT REALLY CLEAR WHAT THIS IS BECAUSE AB USES A FIXED-STEP SIZE. MAYBE THIS
+    IS A CLIP?
+
+MD3-C1
+    NOT REALLY CLEAR WHAT THIS IS BECAUSE AB USES A FIXED-STEP SIZE. MAYBE THIS
+    IS A CLIP?
+
+
+Termination
+"""""""""""
+
+The criteria for terminating the integration. See specific operator for
+details.
+
+
+
+Appearance
+^^^^^^^^^^
+
+Specify appearance settings for the curves. See specific operator for details
+on most settings.
+
+Streamlines vs Pathlines
+""""""""""""""""""""""""
+
+The user may select the integral curve to be based on an instantaneous or
+time-varying vector field producing streamlines or pathlines, respectively. A
+streamline is path drawn by an integrator that uses the same vector field for
+the entire integration. A pathline uses the vector field that is in-step with
+the integrator, so as the integrator steps through time, it draws from the
```

Comment:
`draws` might not be the best word here..`extracts` or `pulls from` or some other synonym to avoid confusioin with visual act of drawing something.